def raise_exception_msg(message=""):
    raise NameError("This is a Name Error")
